classes/PodsAPI.php

```diff
@@ -2452,7 +2475,7 @@ public function save_pod( $params, $sanitized = false, $db = true ) {
 			&& in_array( $pod['type'], array(
 		     	'post_type',
 				'taxonomy'
-			) )
+			), true )
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: c787c41e04fffd73c5708a2cd3b246c0</sub>